picard/ui/util.py

```diff
@@ -89,6 +89,29 @@ def __init__(self, *args):
         for view in self.findChildren((QtWidgets.QListView, QtWidgets.QTreeView)):
             if isinstance(view.model(), QtGui.QFileSystemModel):
                 view.setSelectionMode(QtWidgets.QAbstractItemView.SelectionMode.ExtendedSelection)
+        self.set_sidebar_locations()
+
+    def set_sidebar_locations(self):
+        # Allow access to all mounted drives in the sidebar
+        root_volume = "/"
+        volume_paths = []
+        for volume in QtCore.QStorageInfo.mountedVolumes():
+            if volume.isValid() and volume.isReady():
+                path = volume.rootPath()
+                if volume.isRoot():
+                    root_volume = path
+                else:
+                    if not IS_LINUX or (path.startswith("/media/") or path.startswith("/mnt/")):
+                        volume_paths.append(path)
+        sidebar_urls = [
+            QtCore.QUrl.fromLocalFile(root_volume),
+            QtCore.QUrl.fromLocalFile(QtCore.QDir.homePath()),
+            QtCore.QUrl.fromLocalFile(QtCore.QStandardPaths.writableLocation(QtCore.QStandardPaths.StandardLocation.MusicLocation))
```

Comment:
According to https://doc.qt.io/qt-6/qstandardpaths.html#writableLocation `QtCore.QStandardPaths.writableLocation` could return an empty string, though `QtCore.QStandardPaths.StandardLocation.MusicLocation` is said to default to a sensible location, perhaps safer to check the result and not add it if empty.

See https://github.com/phw/picard/pull/21